lib/exchange_rate.py

```diff
@@ -363,17 +367,18 @@ def set_fiat_address_config(self, b):
 
     def get_currency(self):
         '''Use when dynamic fetching is needed'''
-        return self.config.get("currency", "EUR")
+        return self.config.get("currency", DEFAULT_CURRENCY)
 
     def config_exchange(self):
-        return self.config.get('use_exchange', 'Kraken')
+        return self.config.get('use_exchange', DEFAULT_EXCHANGE)
 
     def show_history(self):
         return self.is_enabled() and self.get_history_config() and self.ccy in self.exchange.history_ccys()
 
     def set_currency(self, ccy):
         self.ccy = ccy
-        self.config.set_key('currency', ccy, True)
+        if self.get_currency() != ccy:
```

Comment:
Nice little performance tune-up I guess?

Reply:
Ah ha -- great approach to do that.  That's pretty cool. Makes even more sense then. 👍